tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -5025,6 +5025,7 @@ SpirvEmitter::processIntrinsicMemberCall(const CXXMemberCallExpr *expr,
     break;
   case IntrinsicOp::MOP_CalculateLevelOfDetail:
     retVal = processTextureLevelOfDetail(expr, /* unclamped */ false);
+    break;
```

Comment:
This also looks like a behavior change. Probably a bug fix, but a change.

Reply:
As before, I think it's correct. @Keenuts @cassiebeckley?

Reply:
This changed the output of a test, but since the test was expecting two LOD calculations where it only asked for one, I feel good about it. That said, it looks like the generated results were the same suggesting no distinction between `CalculateLevelofDetail` and `CalculateLevelofDetailClamped`. Which isn't right.

Reply:
TL;DR: thanks for fixing! Break is needed!

Wasn't aware of this at all. Looked into the SPIR-V spec, and I have trouble parsing it:
```
The first component of the result contains the mipmap array layer.
The second component of the result contains the implicit level of detail relative to the base level.
```

The initial commit added the implicit fallthrough, which is surprising, but the test code seems to match what glslang is doing, aka taking the second component of the result: https://github.com/KhronosGroup/glslang/issues/1423

Looking at tests at ce495eb611be06cb7249c174ea93a61ec163eaeb, it does seem to work as expected at least from the test PoV but I believe it didn't. The tests looks for positive, not negative presence. Hence because both were present, even if it was dead code.
Thanks for fixing!